internal/apischema/awsbedrock/awsbedrock.go

```diff
@@ -5,44 +5,355 @@ import (
 	"strings"
 )
 
-// ConverseRequest is defined in the AWS Bedrock API:
-// https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_Converse.html#API_runtime_Converse_RequestBody
-type ConverseRequest struct {
-	Messages []Message `json:"messages,omitempty"`
+const (
+	// StopReasonEndTurn is a StopReason enum value
```

Comment:
This is a super style nit but I would really like any line comments ending with period to complete the sentence. Could you add period here and elsewhere?
```suggestion
	// StopReasonEndTurn is a StopReason enum value.
```